packages/malloy/src/malloy.ts

```diff
@@ -1131,7 +1132,9 @@ export class PreparedResult {
    * @return The query timezone.
    */
   public get queryTimezone(): string | undefined {
```

Comment:
The result structdef of a query should already have the correct timezone in it, and more things than the renderer will break of that isn't true.

The logic to make that happen, and this is in `lang/model/` so it is a deep dive for me to to figure out, is that the timezone of a query is known when the query is "loaded" and it is the last one in the list of

```
[
  The structdef that is the source for the query,
  ... each query operation in the query pipeline
]
```